h/static/scripts/controllers/character-limit-controller.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+function CharacterLimitController(element) {
+  var maxlength = parseInt(element.getAttribute('data-maxlength'));
```

Comment:
The standard way to read data properties is via the `dataset` attribute: `element.dataset.maxlength` (note that hyphens in the name are converted to `camelCase`)